package/MDAnalysis/core/AtomGroup.py

```diff
@@ -1074,6 +1076,8 @@ def __add__(self, other):
         if not isinstance(other, (Atom, AtomGroup)):
             raise TypeError('Can only concatenate Atom or AtomGroup (not "{0}") to'
                             ' AtomGroup'.format(other.__class__.__name__))
+        if (self and other) and (not self.universe is other.universe):
```

Comment:
AtomGroups are just lists of Atoms really, and I don't want to quote pep8 on you, but there is a pep between 7 and 9 that says:
```
For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

Yes: if not seq:
     if seq:

No: if len(seq)
    if not len(seq)
```